src/js/themes/base.js

```diff
@@ -959,6 +967,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           pad: { horizontal: 'small', vertical: 'xsmall' },
         },
       },
+      // orderedSizes: [undefined],
```

Comment:
```suggestion
      // orderedSizes: undefined,
```
This is more how we annotate base.js